### PR TITLE
Point all past runs to archive branches

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

**Comment by encukou** [comment]
The archive branches have the `archive/` prefix, so we don't touch them by mistake.
I've set up GitHub branch protection so that only owners/maintainers can push to them.

File: runs/2017/mipyt-kam/link.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-repo: https://github.com/cvut/naucse.python.cz.git
-branch: b171
+repo: https://github.com/pyvec/naucse.python.cz/
```
Comment (hroncok) [inline runs/2017/mipyt-kam/link.yml]: why the change in URL?

**Comment by hroncok** [comment]
Do we want this? I mean, people delete branches, apparently, but I was fine having all branches at my own forks.

**Comment by encukou** [comment]
There was a Slack discussion on this recently.
I think we do want this for most other forks.
I think the branch namespace is a good idea: people tend to make branches for quick PRs under `pyvec`; it should be clear which ones can be deleted.
As to your fork, will this inconvenience you? If not, might as well do the same thing everywhere -- it might help with automating this later.

See https://github.com/pyvec/naucse.python.cz/issues/419 for the 2017/2018 archival.

File: runs/2017/mipyt-kam/link.yml (hunk 1 above)
Comment (encukou) [reply runs/2017/mipyt-kam/link.yml]: Hmm, I didn't think about this, to be honest – I just added the pyvec URL everywhere, and it turns out I changed the existing repo names. Is it a problem?

Generally I like `/` rather than `.git` a bit better since you can just add e.g. `pulls/` to the end and get a valid URL.

Comment (hroncok) [reply runs/2017/mipyt-kam/link.yml]: I don't care either way, I was just curious.

**Comment by hroncok** [comment]
> As to your fork, will this inconvenience you?

Not really, I don't change the past branches at all.

**Comment by encukou** [comment]
So, is it good to go?

**Comment by hroncok** [comment]
The idea? Yes. But I haven't checked if the branches match the previous content nor whether the change is conplete. Do you want me to?

**Comment by encukou** [comment]
No, unless you want to :)
I'll check the output and merge if it's similar enough.

**Comment by encukou** [comment]
The diff was [too much for meld](https://bugzilla.redhat.com/show_bug.cgi?id=1788103) at one point, but it looks OK:

* "Canonical" courses have code highlighting differences due to newer pygments. I'd say they're improvements – e.g. `async` is a keyword; `print` is not.
* `2018/czechitas-ostrava-jaro` has the same differences (see https://github.com/pyvec/naucse.python.cz/issues/578)
* iCal formatting changed somewhat with the new version of `ics`, but AFAICS the information is the same.
* Edit URLs changed to the archives, of course
